platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -333,106 +318,114 @@ also measure users' experiences, including:
 
 ## Capabilities of platforms
 
-Having defined key attributes of a platform and platform teams, following are
-capabilities required by typical digital projects which should be considered for
-inclusion in a platform.
-
-The goal of this list is to guide platform teams in determining what should be
-included in their platform.
-
-1. Web portal for provisioning and observing capabilities
-1. APIs (and CLIs) for automatically provisioning capabilities on demand
-1. "Golden path" templates and docs enabling optimal use of capabilities
-1. Automation for building and testing services and products
-1. Automation for delivering and verifying services and products
-1. Services in support of product research and development: hosted IDEs, remote
-   connection tools
-1. Instrumentation and dashboards to enable observability of services and
-   products
-1. Infrastructure services: compute runtime, programmable network, block storage
-1. Data services: databases, caches, object stores
-1. Messaging services: brokers, queues, event fabrics
-1. Identity and secret management services: service and user identity and
+As described earlier, a platform for cloud-native computing offers and composes
+capabilities and services from many supporting providers. These providers may be
+other teams within the same enterprise or third parties like "cloud" service
+providers. Platforms enable use of the capabilities and services from these
+providers by wrapping them with consistent web portals, documentation, code
+templates, and programmable APIs and tools. Of note, platforms bridge from
+underlying capability providers to platform users like application developers
+while implementing and enforcing desired practices for security, performance and
+consistent experience.
```

Comment:
I would love to see the last sentence get a bit more punch than "of note" lends. IMO this is absolutely a key differentiator for successful platforms.

Also added a small added bonus of cost as I think finops has become quite important for people these days.

```suggestion
templates, and programmable APIs and tools. This wrapping of services also 
provides an opportunity for the organisation to provide capabilities while
implementing and enforcing desired practices for security, performance, 
cost optimisation and consistent experience.
```

Reply:
I'm going to split off the last sentence into another paragraph and precede with some transition. Will also add cost governance :+1: